modules/xmpp/JingleSession.js

```diff
@@ -56,14 +56,21 @@ function JingleSession(me, sid, peerjid, connection,
      * @type {JingleSessionState}
      */
     this.state = null;
+
+    /**
+     * The RTC service instance
+     * @type {RTC|null}
```

Comment:
It depends whether or not it is expected to handle the null case. Here RTC will be initialised later (not in the constructor), but I would say that the instance should not be used prior to that, so I agree that "|null" can be removed (it is not expected to handle null).